cypress/support/index.d.ts

```diff
@@ -1,9 +1 @@
 import '@mate-academy/cypress-tools/support';
```

Comment:
The entire global declaration block has been removed which might cause problems if you are using the byDataCy custom command in your Cypress tests. If needed, you should add it back.